community-operators/microcks/microcks-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,206 @@
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - '*'
+        - apiGroups:
+          - extensions
+          resources:
+          - ingresses
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - microcks.github.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+      deployments:
+      - name: microcks-ansible-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: microcks-ansible-operator
+          template:
+            metadata:
+              labels:
+                name: microcks-ansible-operator
+            spec:
+              serviceAccountName: microcks-ansible-operator
+              containers:
+                - name: microcks-ansible-operator
+                  image: quay.io/microcks/microcks-ansible-operator
+                  imagePullPolicy: Always
+                  env:
+                    - name: WATCH_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.namespace
+                    - name: POD_NAME
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.name
+                    - name: OPERATOR_NAME
+                      value: "microcks-ansible-operator"
+  customresourcedefinitions:
+    owned:
+    - name: microcksinstalls.microcks.github.io
+      version: v1alpha1
+      kind: MicrocksInstall
+      displayName: MicrocksInstall
+      description: Represents a Microcks installation
+      resources:
+        - kind: Service
+          name: ''
+          version: v1
+        - kind: Deployment
+          name: ''
+          version: v1
+        - kind: Pod
+          name: ''
+          version: v1
+        - kind: ConfigMap
+          name: ''
+          version: v1
+        - kind: Secret
+          name: ''
+          version: v1
+        - kind: PersistentVolumeClaim
+          name: ''
+          version: v1
+      specDescriptors:
+        - description: Name of your MicrocksInstall
+          displayName: Name
+          path: name
+          x-descriptors:
+            - 'urn:alm:descriptor:com.tectonic.ui:label'
+        - description: Version of Microcks to install
+          displayName: Version
+          path: version
+          x-descriptors:
+            - 'urn:alm:descriptor:com.tectonic.ui:label'
```

Comment:
nit: Newline

Reply:
Added in commit caddfc74bd5023fc82628e70d9ca8bd08a643bc8